x/crosschain/types/status.go

```diff
@@ -16,9 +16,12 @@ func AllStatus() []CctxStatus {
 	}
 }
 
+// empty msg does not overwrite old status message
 func (m *Status) ChangeStatus(ctx *sdk.Context, newStatus CctxStatus, msg, logIdentifier string) {
 	oldStatus := m.Status
-	m.StatusMessage = msg
+	if len(msg) > 0 {
```

Comment:
FYI @kingpinXD 
add a easy way to change status without changing the status message; useful in retaining error messages when useful.